inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/release.adoc

```diff
@@ -0,0 +1,101 @@
+= Releasing INCEpTION
+
+== Prerequisites
+
+When deploying to an Artifactory instance, make sure that the UKP Maven Repository is with your
+user credentials are added to your machines `settings.xml`. In our case, the repository
+`ukp-oss-releases` has to be listed with a user and password that belongs to the `trusted-user`
+group. See https://dkpro.github.io/dkpro-core/pages/setup-maven[DKPro Core - Setting up Maven repository]
+for more information on how to do this.
+
+== Release steps
+
+[arabic]
+. Create a release issue in the INCEpTION Github issue tracker
```

Comment:
I am a bit worried about this documentation getting out of sync with the issue tracker template. In fact, it is already out of sync because the template doesn't say anything about merging the release branch into master. Any idea how we could avoid this?

Reply:
For me, the things that are important for this guide and also mentioned in the issue template are in `Local` and `Docker`, so I do not think that they are out of sync

Reply:
E.g. the templates does not include a checkbox for merging changes from the maintenance branch into the master branch.